app/src/main/res/drawable/ic_external_link_green_16dp.xml

```diff
@@ -0,0 +1,14 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="16dp"
+    android:height="16dp"
+    android:viewportWidth="16"
+    android:viewportHeight="16">
+  <path
+      android:pathData="M13,13v-2c0,-0.552 0.448,-1 1,-1s1,0.448 1,1v2c0,1.105 -0.895,2 -2,2H3c-1.105,0 -2,-0.895 -2,-2V3c0,-1.105 0.895,-2 2,-2h2c0.552,0 1,0.448 1,1s-0.448,1 -1,1H3v10h10z"
+      android:fillColor="#008C73"
```

Comment:
Can't we reuse `ic_link_green_24dp.xml`? If not, can you set the `fillColor` to `@color/primary` so it is correct in Dark mode?